helm-charts/charts/kube-starrocks/values.yaml

```
@@ -67,74 +65,64 @@ starrocksFESpec:
     loadbalancerIP: ""
     # add annotations for fe service.
     annotations: {}
-
-  # contains the secret name
+  # imagePullSecrets allows you to use secrets to pull images for pods.
   imagePullSecrets: []
-  # - name: "image-pull-secret"
-
+    # - name: "image-pull-secret"
   # serviceAccount for fe access cloud service.
   serviceAccount: ""
-
   # If specified, the pod's nodeSelector，displayName="Map of nodeSelectors to match when scheduling pods on nodes"
   # Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
   nodeSelector: {}
-  #  beta.kubernetes.io/arch: amd64
-  #  beta.kubernetes.io/os: linux
-
+    # beta.kubernetes.io/arch: amd64
+    # beta.kubernetes.io/os: linux
```

```suggestion
    # kubernetes.io/arch: amd64
    # kubernetes.io/os: linux
```

should be no `beta` prefix?

---

yes, no beta prefix. And there are three places that should be fixed like this. I will change it myself.

---

enn, my local cluster build by kind has both labels（kubectl get nodes -oyaml）, like this:
```
beta.kubernetes.io/arch: arm64
beta.kubernetes.io/os: linux
kubernetes.io/arch: arm64
kubernetes.io/os: linux
```
whatever, I remove the `beta` prefix.